lib/util/ImpCentralApiHelper.js

```diff
@@ -220,6 +204,26 @@ class ImpCentralApiHelper {
         });
     }
 
+    _refreshAccessToken() {
+        if (!this._authConfig.refreshToken && !this._authConfig.loginKey) {
+            return Promise.reject(new Errors.AccessTokenExpiredError(this._authConfig));
+        }
+        else {
+            return this._renewAccessToken()
+                .then(result => {
+                    this._authConfig.setAccessToken(result.access_token, result.expires_at);
+                    this._impCentralApi.auth.accessToken = this._authConfig.accessToken;
+                    return this._authConfig.save();
+                }).catch(error => {
+                    if (error instanceof Errors.EntityNotFoundError ||
+                        this._isEntityNotFoundError(error)) {
+                        return Promise.reject(new Errors.RefreshTokenError(this._authConfig));
+                    }
+                    return Promise.reject(error);
```

Comment:
Does it mean we just exit impt with an error message?

Reply:
Yes, if renew access token operation was failed, impt exit with an error

Reply:
ok sounds reasonable